packages/platforms/rivet/src/actor_driver.ts

```diff
@@ -7,6 +7,7 @@ export interface ActorDriverContext {
 
 export class RivetActorDriver implements ActorDriver {
 	#ctx: ActorContext;
+	#alarm: { timeout: ReturnType<typeof setTimeout>; timestamp: number } | null = null;
```

Comment:
It's not completely clear to me if the rivet driver is instantiated once per actor like the cloudflare driver is once per durable object. I've implemented this assuming that it is.